src/main/java/com/google/sps/data/TimeZone.java

```diff
@@ -0,0 +1,103 @@
+  MEDT("Middle East Standard Time", 3, "(UTC+02:00) Beirut"),
+  EST("Egypt Standard Time", 2, "(UTC+02:00) Cairo"),
+  SDT("Syria Standard Time", 3, "(UTC+02:00) Damascus"),
+  EEDT("E. Europe Standard Time", 3, "(UTC+02:00) E. Europe"),
+  FDT("FLE Standard Time", 3, "(UTC+02:00) Helsinki, Kyiv, Riga, Sofia, Tallinn, Vilnius"),
+  TDT("Turkey Standard Time", 3, "(UTC+03:00) Istanbul"),
+  JDT("Israel Standard Time", 3, "(UTC+02:00) Jerusalem"),
+  LST("Libya Standard Time", 2, "(UTC+02:00) Tripoli"),
+  KST("Kaliningrad Standard Time", 3, "(UTC+02:00) Kaliningrad"),
+  EAST("E. Africa Standard Time", 3, "(UTC+03:00) Nairobi"),
+  MSK("Moscow Standard Time", 3, "(UTC+03:00) Moscow, St. Petersburg, Volgograd, Minsk"),
+  SAMT("Samara Time", 4, "(UTC+04:00) Samara, Ulyanovsk, Saratov"),
+  IDT("Iran Standard Time", 4.5, "(UTC+03:30) Tehran"),
+  GET("Georgian Standard Time", 4, "(UTC+04:00) Tbilisi"),
+  WAST("West Asia Standard Time", 5, "(UTC+05:00) Ashgabat, Tashkent"),
+  YEKT("Yekaterinburg Time", 5, "(UTC+05:00) Yekaterinburg"),
+  PKT("Pakistan Standard Time", 5, "(UTC+05:00) Islamabad, Karachi"),
+  IST("India Standard Time", 5.5, "(UTC+05:30) Chennai, Kolkata, Mumbai, New Delhi"),
+  SLST("Sri Lanka Standard Time", 5.5, "(UTC+05:30) Sri Jayawardenepura"),
+  CAST("Central Asia Standard Time", 6, "(UTC+06:00) Nur-Sultan (Astana)"),
+  SAST("SE Asia Standard Time", 7, "(UTC+07:00) Bangkok, Hanoi, Jakarta"),
+  NCAST("N. Central Asia Standard Time", 7, "(UTC+07:00) Novosibirsk"),
+  CST("China Standard Time", 8, "(UTC+08:00) Beijing, Chongqing, Hong Kong, Urumqi"),
+  NAST("North Asia Standard Time", 8, "(UTC+08:00) Krasnoyarsk"),
+  MPST("Singapore Standard Time", 8, "(UTC+08:00) Kuala Lumpur, Singapore"),
+  UST("Ulaanbaatar Standard Time", 8, "(UTC+08:00) Ulaanbaatar"),
+  NAEST("North Asia East Standard Time", 8, "(UTC+08:00) Irkutsk"),
+  JST("Japan Standard Time", 9, "(UTC+09:00) Osaka, Sapporo, Tokyo"),
+  ACST("AUS Central Standard Time", 9.5, "(UTC+09:30) Darwin"),
+  AEST("AUS Eastern Standard Time", 10, "(UTC+10:00) Canberra, Melbourne, Sydney"),
+  WPST("West Pacific Standard Time", 10, "(UTC+10:00) Guam, Port Moresby"),
+  YST("Yakutsk Standard Time", 9, "(UTC+09:00) Yakutsk"),
+  CPST("Central Pacific Standard Time", 11, "(UTC+11:00) Solomon Is., New Caledonia"),
+  NZST("New Zealand Standard Time", 12, "(UTC+12:00) Auckland, Wellington"),
+  U("UTC+12", 12, "(UTC+12:00) Coordinated Universal Time+12"),
+  FST("Fiji Standard Time", 12, "(UTC+12:00) Fiji"),
+  KDT("Kamchatka Standard Time", 13, "(UTC+12:00) Petropavlovsk-Kamchatsky - Old"),
+  TST("Tonga Standard Time", 13, "(UTC+13:00) Nuku'alofa"),
+  SST("Samoa Standard Time", 13, "(UTC+13:00) Samoa");
+
+  private String title;
+  private double utcOffset;
+  private String longText;
+
+  private TimeZone(String title, double utcOffset, String longText) {
+    this.title = title;
+    this.utcOffset = utcOffset;
+    this.longText = longText;
+  }
```

Comment:
its probably easier to use a java date time library than to write all this (something like joda data time)